### DISCUSSION TITLE
Custom Reponse

### DISCUSSION THREAD

**Discussion by Eyup-Devop** [corazawaf/coraza#952] [Q&A]
hello, 
I want to send to client custom page for blocked request, is there a way to do it 
txhttp.WrapHandler(waf, ...)

**Comment by M4tteoP** [comment]
Hey @Eyup-Devop, I don't think so. You would have to tweak the middleware everywhere (e.g. https://github.com/corazawaf/coraza/blob/b887a580da2c8d76e0abbf8f5e029735d0f89313/http/middleware.go#L146-L147) an interruption is enforced. Currently, it is just setting the status code su 403. Feel free to open an issue about it to track the feature request if native support might be useful

**Comment by jcchavezs** [comment]
Can't you do a redirect?

**Reply by Eyup-Devop** [reply]
how can I do redirect?

**Comment by jcchavezs** [comment]
Try https://coraza.io/docs/seclang/actions/#redirect
